core/src/queue.rs

```diff
@@ -424,18 +416,44 @@ mod tests {
     }
 
     #[test]
+    #[ignore = "Multisignature is not working for now. See #2595"]
```

Comment:
So I believe you have fixed the test? We can unignore it?

Reply:
@Arjentix not until #3043, which [does unignore](https://github.com/hyperledger/iroha/pull/3043/commits/98b8242268abbeaa5a77906308d15baf842b5123#diff-e6b9f4fffa1a86e9147aab986d762b234878cfa7a1900ee45487d5d9ca91a156L419) it.